### PR TITLE
Added PyTorch tutorial

### PR DESCRIPTION
**Pull request by sahehb**

### CODE REVIEW

File: mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+![Show Enclosing Folder](/images/tutorials/thirdparty/pytorch_example3_2.png "Show Enclosing Folder")
+
+The explorer opens showing the files of your macro module. Copy the *\*.mlab* file somewhere you can remember.
+
+### Create the macro module
+Open the the Project Wizard via {{< menuitem "File" "Run Project Wizard">}} and select *Macro Module*. Click *Run Wizard*.showing the internal network.
+
+![Project Wizard](/images/tutorials/thirdparty/pytorch_example3_3.png "Project Wizard")
+
+Define module properties as seen below, you can chose any name you like. Click *Next*.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md:36]: Define the module properties as shown below, though you can chose your own name.

File: mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+![Module Properties](/images/tutorials/thirdparty/pytorch_example3_4.png "Module Properties")
+
+Define macro module properties and select the copied *\*.mlab* file. Make sure to add a Python file and click *Next*.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md:40]: the module properties

File: mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+![Macro Module Properties](/images/tutorials/thirdparty/pytorch_example3_5.png "Macro Module Properties")
+
+Leave the module field reference as is and click *Create*. Close Project Wizard and select {{< menuitem "Extras" "Reload Module Database (Clear Cache)">}}.
+
+### Re-use script and Python code
+Open the script file of the `WebCamTest` module and copy to your new PyTorch module. The result should be something like this:
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty/pytorch/pytorchexample3.md:47]: ..and copy the contents...

**Comment by sahehb** [comment]
any more changes requested here? :)

**Review by aliahcodes — APPROVED** [review]

**Pull request merged**